tools/ldcontext_terms_mappings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "AgriCrop": "Agri-Crop",
+    "AgriGreenhouse": "Agri-Greenhouse",
+    "AgriParcelOperation": "Agri-Parcel-Operation",
+    "AgriParcelRecord": "Agri-Parcel-Record",
+    "AgriParcel": "Agri-Parcel",
+    "AgriPest": "Agri-Pest",
+    "AgriProductType": "Agri-Product-Type"
+}
```

Comment:
nit: new line needed

Reply:
Fixed in 8933b1ef27836a637fd56c3530e5b226b30a2fdb